style.css

```diff
@@ -0,0 +1,902 @@
+}
+
+.navigation.active .bar:nth-child(3) {
+  transform: translateY(-9px) rotate(-45deg);
+}
+
+.bg {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  height: fit-content;
+  width: 100%;
+  background-image: url("image/shapes.png");
+  padding-top: 20%;
+  padding-left: 3%;
+  padding-bottom: 20%;
+  background-attachment: local;
+  background-size: cover;
+  background-repeat: no-repeat;
+  gap: 10%;
+}
+
+.bg h1 {
+  height: fit-content;
+  font-weight: 800;
+  font-size: 38px;
+  align-items: center;
+  color: #172b4d;
+  margin: 6%;
+  margin-bottom: 3%;
+  width: 320px;
+}
+
+.bg p {
+  font-weight: 400;
+  height: fit-content;
+  margin-left: 6%;
+  margin-right: 6%;
+  margin-bottom: 3%;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.bg .connect {
+  font-size: 16px;
+  line-height: 13px;
+  height: fit-content;
+  margin-bottom: 3%;
+  color: #6070ff;
+}
+
+.bg .social-1 {
+  padding-left: 6%;
+  display: flex;
+  gap: 4px;
+  list-style-type: none;
+}
+
+.skills a img {
+  padding: 2%;
+  width: auto;
+}
+
+.social-1 li a img:hover {
+  transform: scale(1.2);
+  transition: 0.5s;
+}
+
+@media only screen and (min-width: 768px) {
+  .top-container {
+    display: grid;
+    background: #fff;
+    height: fit-content;
+    width: 100%;
+    align-self: center;
+    align-items: center;
+    border-bottom-left-radius: 100px;
+  }
+
+  .navigation {
+    display: none;
+    cursor: pointer;
+    transition: ease-in;
+  }
+
+  nav ul {
+    display: flex;
+  }
+
+  .bg {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    height: fit-content;
+    width: 100%;
+    background-image: url("image/Header-bg.png");
+    padding-top: 12%;
+    padding-left: 3%;
+    padding-bottom: 12%;
+    background-attachment: local;
+    background-size: contain;
+    background-repeat: no-repeat;
+    gap: 10%;
+  }
+
+  .menu-list {
+    display: block;
+    margin-right: 4%;
+    transition-property: width;
+    transition-duration: 2s;
+    transition-timing-function: linear;
+    transition-delay: 1s;
+  }
+
+  .menu-list ul {
+    gap: 12px;
+  }
+
+  .menu-list ul li {
+    list-style: none;
+    display: inline-block;
+  }
+
+  .menu-list li a {
+    text-decoration: none;
+    list-style: none;
+    display: inline-block;
+  }
+
+  .bg h1 {
+    height: fit-content;
+    font-weight: 800;
+    font-size: 38px;
+    align-items: center;
+    color: #172b4d;
+    margin: 6%;
+    margin-bottom: 1%;
+    width: 450px;
+  }
+
+  .bg p {
+    font-weight: 400;
+    font-size: 18px;
+    margin-bottom: 1%;
+  }
+
+  .bg .connect {
+    margin-bottom: 1%;
+  }
+
+  .bg .social-1 {
+    padding-left: 6%;
+    display: flex;
+    gap: 4px;
+    list-style-type: none;
+  }
+}
+
+.grid-container {
+  display: grid;
+  width: 100%;
+  padding-bottom: 6%;
+  height: 2600px;
+  justify-content: center;
+  align-content: center;
+  row-gap: 2%;
+  align-items: center;
+}
```

Comment:
- [ ] Good job trying to use `<grid>` for the works section 👍🏽 . THen grid seems not to take effect because you didn't use the grid properly. I believe this is causing your projects card to overlap each other and even overlap the headline section making the app not look great. Kindly look into this. 👍🏽 

Learn how to properly use grid []([here](https://css-tricks.com/snippets/css/complete-guide-grid/))
![image](https://user-images.githubusercontent.com/63473584/213788468-98d7ec08-3239-423e-8234-4f42fba19403.png)

![image](https://user-images.githubusercontent.com/63473584/213788530-0a3444bb-b5f2-4b25-9be4-315bd02d4b10.png)
 
Make sure your design looks like the Figma design 👍🏽